<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorView</class>
 <widget class="QWidget" name="DoctorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>医生管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="searchLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>搜索：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSearch">
       <property name="placeholderText">
        <string>请输入医生姓名或工号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btSearch">
       <property name="text">
        <string>搜索</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cmbDepartmentFilter">
       <property name="placeholderText">
        <string>按科室筛选</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btAdd">
       <property name="text">
        <string>添加</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/add.png</normaloff>:/icons/add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btDelete">
       <property name="text">
        <string>删除</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btEdit">
       <property name="text">
        <string>编辑</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>刷新</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="statLayout">
     <item>
      <widget class="QLabel" name="lblStats">
       <property name="text">
        <string>总计：0 | 在职：0 | 休假：0 | 退休：0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>导出</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/export.png</normaloff>:/icons/export.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
